.vscode/settings.json

```diff
@@ -1,11 +1,4 @@
-// Place your settings in this file to overwrite default and user settings.
 {
-    // Columns at which to show vertical rulers
-    "editor.rulers": [100],
-
-    // Controls after how many characters the editor will wrap to the next line. Setting this to 0 turns on viewport width wrapping (word wrapping). Setting this to -1 forces the editor to never wrap.
```

Comment:
This is something that should be configured at the User level, not the Workspace level. 

HOWEVER... you bring up a good point. The way this is configured now, there will always be no rulers in this Workspace, since the `editor.rulers` value is defined as `[]`. I'll fix that so you can set your VSCode User settings to whatever you like.

Reply:
So if you see that `"editor.rulers": [],` line in any of the `.vscode/settings.json` files of our repos, make a note of it, since it should be removed.

Reply:
fixed in commit [ccc2fa7 - "fix workspace settings"](https://github.com/blackbaud/skyux2-docs/pull/1076/commits/ccc2fa75b9bcbca115862e59a5222628f75026e5)